packages/destination-actions/src/destinations/actions-facebook-conversions/purchase/index.ts

```diff
@@ -0,0 +1,66 @@
+import { ActionDefinition, IntegrationError } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { CURRENCY_ISO_CODES } from '../constants'
+import { currency, value, content_name, content_type, contents, num_items, content_ids } from '../fb-capi-properties'
+import { user_data } from '../fb-capi-user-data'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Purchase',
+  description: 'Send a purchase event to FB',
+  fields: {
+    value: { ...value, required: true },
+    currency: { ...currency, required: true },
+    content_name: content_name,
+    content_type: content_type,
+    contents: contents,
+    num_items: num_items,
+    content_ids: content_ids,
+    action_source: {
+      label: 'Action Source',
+      description: 'The action source of the event',
+      type: 'string'
+    },
+    user_data: user_data
+  },
+  perform: (request, { payload, settings }) => {
+    if (!CURRENCY_ISO_CODES.includes(payload.currency)) {
```

Comment:
If you make the `Set` update https://github.com/segmentio/action-destinations/pull/122/files#r699840738

... then this changes to

```suggestion
    if (!CURRENCY_ISO_CODES.has(payload.currency)) {
```